WebDAVClient/Client.cs

```diff
@@ -759,6 +761,24 @@ private static bool TryCreateAbsolute(string uriString, out Uri uriResult)
             }
         }
 
+        /// <summary>
```

Comment:
Casting int? to (int) while it doesn't have a variable returns a exception, therefore I tried to make a little auto detect function for it when it's unset.